openquake/hazardlib/contexts.py

```diff
@@ -74,14 +74,25 @@ def concat(ctxs):
     Concatenate context arrays.
     :returns: list with 0, 1 or 2 elements
     """
-    out, parametric, nonparam = [], [], []
+    out, parametric, parametric_np, nonparam = [], [], [], []
```

Comment:
What does is mean parametric_np? Perhaps a better name would be parametric_negbin?